cpmpy/solvers/minizinc.py

```diff
@@ -184,6 +186,7 @@ def solve(self, time_limit=None, **kwargs):
         import minizinc.error
         try:
             mzn_result = mzn_inst.solve(**mzn_kwargs)
+            self.mzn_result = mzn_result
```

Comment:
Why not directly
`self.mzn_result = mzn_inst.solve(**mzn_kwargs)?`

Reply:
mzn_results gets used a couple of times more in the function, does it make more sense to then use self.mzn_result there in stead?

Reply:
I think so yes